pkg/koordlet/util/container.go

```diff
@@ -78,6 +81,13 @@ func GetContainerCgroupCPUAcctPSIPath(podParentDir string, c *corev1.ContainerSt
 	if err != nil {
 		return PSIPath{}, err
 	}
+	if system.GetCurrentCgroupVersion() == system.CgroupVersionV2 {
```

Comment:
How about adding and implementing the methods `ReadCPUPressure()` `ReadMemoryPressure()` `ReadIOPressure()` (or a single `ReadPSI()`) for the `CgroupReader` interface?